core/current_user_api.php

```diff
@@ -247,6 +247,7 @@ function current_user_get_bug_filter( $p_project_id = null ) {
 				$t_filter = json_decode( $t_token, true );
 			}
 		} else {
+			// TODO @cproensa remove this, remainders of ancient #4609
```

Comment:
Here is some code that probably should be cleaned.
Do not use raw json filter strings as url parameters.
Refactor the tokens get/set to allow multiple filters per user, (and get-by-id)